packages/shared/src/lib/boot.ts

```diff
@@ -87,21 +87,32 @@ export async function getBootData(app: string, url?: string): Promise<Boot> {
     params.append('url', url);
   }
 
-  const res = await fetch(`${apiUrl}/boot${appRoute}?${params}`, {
-    method: 'GET',
-    credentials: 'include',
-    headers: { app, 'Content-Type': 'application/json' },
-  });
-  const result = await res.json();
+  try {
+    const res = await fetch(`${apiUrl}/boot${appRoute}?${params}`, {
+      method: 'GET',
+      credentials: 'include',
+      headers: { app, 'Content-Type': 'application/json' },
+    });
 
-  const features = await decrypt(
-    result.exp.f,
-    process.env.NEXT_PUBLIC_EXPERIMENTATION_KEY,
-    'AES-CBC',
-    128,
-  );
+    if (res.status >= 500) {
+      throw new Error('Server error');
+    }
 
-  result.exp.features = JSON.parse(features);
+    const result = await res.json();
 
-  return result;
+    const features = await decrypt(
+      result.exp.f,
+      process.env.NEXT_PUBLIC_EXPERIMENTATION_KEY,
+      'AES-CBC',
+      128,
+    );
+
+    result.exp.features = JSON.parse(features);
+
+    return result;
+  } catch (e) {
+    throw new Error('Server error');
+  }
```

Comment:
This I think we don't need? It will just throw by itself?

Reply:
Yeah not sure, think I accidently repushed this in 1 commit 🤦